### PR TITLE
[master] Fix spelling errors

### PR DESCRIPTION
**Pull request by mattmoor**
Produced via:
  `github.com/client9/misspell`
/assign ImJasonH vdemeester
/cc ImJasonH vdemeester

### CODE REVIEW

File: pkg/reconciler/pipelinerun/resources/resultrefresolution_test.go [hunk 1]
```diff
@@ -258,7 +258,7 @@ func TestTaskParamResolver_ResolveResultRefs(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			t.Logf("test name: %s\n", tt.name)
 			got, err := extractResultRefsForParam(tt.fields.pipelineRunState, tt.args.param)
-			// sort result ref based on task name to garantee an certain order
+			// sort result ref based on task name to guarantee an certain order
```
Comment (danielhelfand) [inline pkg/reconciler/pipelinerun/resources/resultrefresolution_test.go:261]:
```suggestion
			// sort result ref based on task name to guarantee a certain order
```

**Comment by sbwsg** [comment]
integration's been running for over an hour.  Re-running

/test pull-tekton-pipeline-integration-tests

**Pull request merged**